airplay/start.sh

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 
+#Check for incompatible multi room and client-only setting
+if [[ -n $DISABLE_MULTI_ROOM ]] && [[ $CLIENT_ONLY_MULTI_ROOM == "1" ]]; then
+  echo “DISABLE_MULTI_ROOM and CLIENT_ONLY  cannot be set simultaneously. Ignoring CLIENT_ONLY mode.”
```

Comment:
Change to `CLIENT_ONLY_MULTI_ROOM` for consistency 👌

Reply:
Done!